website/meta/universe.json

```diff
@@ -1,5 +1,72 @@
 {
     "resources": [
+        {
+            "id": "scrubadub_spacy",
+            "title": "scrubadub_spacy",
+            "category": ["pipeline"],
+            "tags": [""],
+            "slogan": "Clean personally identifiable information from dirty dirty text using spaCy.",
+            "description": "scrubadub removes personally identifiable information from text. scrubadub_spacy is an extension that uses spaCy NLP models to remove personal information from text.",
+            "github": "LeapBeyond/scrubadub_spacy",
+            "pip": "scrubadub-spacy",
+            "url": "https://github.com/LeapBeyond/scrubadub_spacy",
+            "code_language": "python",
+            "author": "Thomas Bird",
+            "author_links": {
+                "twitter": "thomasbird",
+                "github": "thomasbird",
+                "website": "http://thomasbird.com"
+            },
+            "code_example": [
+                "import scrubadub, scrubadub_spacy",
+                "scrubber = scrubadub.Scrubber()",
+                "scrubber.add_detector(scrubadub_spacy.detectors.SpacyEntityDetector)",
+                "print(scrubber.clean(\"My name is Philip, I work at Explosion in Berlin, and my eMail is philip@explosion.ai btw. my super secret twitter login is username: philipvollet password: g-dragon180888\"))",
```

Comment:
I think it looks more professional to use fictional text, especially in user-facing code samples.

Reply:
Fix: I used a common name with a fictional company / email (hope it's a better alternative)